specification/storage/data-plane/Microsoft.FileStorage/preview/2020-02-10/file.json

```diff
@@ -2609,7 +2609,7 @@
                         "name": "x-ms-content-length",
                         "x-ms-client-name": "fileContentLength",
                         "in": "header",
-                        "description": "Specifies the maximum size for the file, up to 1 TB.",
```

Comment:
Just checking that this change is intended?

Reply:
Yes. With 2020-02-10 service version, the largest file size supported is 4TB now.